.github/workflows/playwright.yml

```diff
@@ -0,0 +1,51 @@
+name: playwright
+on:
+    # run on push but only for the main branch
+    push:
+        branches:
+            - main
+    # run for every pull request
+    pull_request: {}
+jobs:
+    test:
+        timeout-minutes: 60
+        runs-on: ubuntu-latest
+        steps:
+            - name: 🛑 Cancel Previous Runs
+              uses: styfle/cancel-workflow-action@0.11.0
+              with:
+                  access_token: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: ⬇️ Checkout repo
+              uses: actions/checkout@v3
+              with:
+                  fetch-depth: 0
+
+            - name: 🤝 Set Node version from .nvmrc
```

Comment:
We should probably use this in more of our workflows since we always have the .nvmrc file anyways!

Reply:
Yeah it's great having our Node.js version defined in one place (.nvmrc) and having all other things like GitHub Actions read from it.